subsys/bluetooth/host/conn.c

```diff
@@ -416,18 +416,18 @@ static struct bt_conn_tx *conn_tx_alloc(void)
 int bt_conn_send_iso_cb(struct bt_conn *conn, struct net_buf *buf,
 			bt_conn_tx_cb_t cb, bool has_ts)
 {
-	int err = bt_conn_send_cb(conn, buf, cb, NULL);
-
-	if (err) {
-		return err;
-	}
-
 	/* Necessary for setting the TS_Flag bit when we pop the buffer from the
 	 * send queue.
 	 * Size check for the user_data is already done in `bt_conn_send_cb`.
```

Comment:
```suggestion
```
This is no longer the case. It was actually the reason for the hack of putting the setting of `iso_has_ts` below the call to `bt_conn_send_cb`. (To save some code ROM.) Should add a copy of the check from inside `bt_conn_send_cb` to this function?